modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -12,48 +12,60 @@
 /datum/job/ai
 	alt_titles = list(
 		"AI",
+		"Automated Overseer",
 		"Station Intelligence",
-		"Automated Overseer"
 	)
 
 /datum/job/assistant
 	alt_titles = list(
 		"Assistant",
-		"Civilian",
-		"Tourist",
+		"Artist",
 		"Businessman",
 		"Businesswoman",
-		"Trader",
+		"Civilian",
 		"Entertainer",
 		"Freelancer",
-		"Artist",
-		"Off-Duty Staff",
+		"Tourist",
+		"Trader",
 		"Off-Duty Crew",
+		"Off-Duty Staff",
 	)
 
 /datum/job/atmospheric_technician
 	alt_titles = list(
 		"Atmospheric Technician",
-		"Life Support Technician",
 		"Emergency Fire Technician",
 		"Firefighter",
+		"Life Support Technician",
 	)
 
 /datum/job/barber
 	alt_titles = list(
 		"Barber",
+		"Aethestician",
+		"Colorist",
 		"Salon Manager",
 		"Salon Technician",
 		"Stylist",
-		"Colorist",
 	)
 
 /datum/job/bartender
 	alt_titles = list(
 		"Bartender",
-		"Mixologist",
-		"Barkeeper",
 		"Barista",
+		"Barkeeper",
+		"Mixologist",
+	)
+
+/datum/job/bitrunner
+	alt_titles = list(
+		"Bitrunner",
+		"Bitdomain Technician",
```

Comment:
Honestly I'm not really sold on the Bitdomain Technician. It's really mouthy and doesn't really fit the role either - they're not technicians, they don't maintain the servers. They're raiding virtual servers (honestly from a lore standpoint IDK if this is even legal now that I think about it...)

Same thoughts on Pod Jockey but for different reasons - while it's fitting it just sounds a bit gross/unprofessional. There's no reason an official station ID would call them it

Also Junior Runner needs to be Junior Bitrunner (bc normal running isn't what they do lol)

Reply:
This was a headmin suggestion so i'll leave it up to dem. Pod Jockey is poggers tho u cant tell me what 2 do >:(

Reply:
Eugh! Ech! Ptooey!!